ingest/importer/spreadsheet/ingest_workbook.py

```diff
@@ -26,6 +28,14 @@ def get_worksheet(self, worksheet_title):
 
         return None
 
+    def select_importable_worksheets(self, worksheet_titles: List[str]):
+        worksheets = []
+        for title in worksheet_titles:
+            worksheets.extend([IngestWorksheet(worksheet) for worksheet in self.workbook.worksheets
+                               if (title not in SPECIAL_TABS) and (title in worksheet.title or
+                                                                   title.lower() in worksheet.title.lower())])
```

Comment:
Could we remove `title in worksheet.title` from the conditions, please?
I think this is already covers the case: `title.lower() in worksheet.title.lower()`

Reply:
Good catch! Thanks!